docs/en/sql-reference/functions/regular-functions-index.md

```diff
@@ -44,6 +44,7 @@ title: 'Regular Functions'
 | [Distance](/sql-reference/functions/distance-functions) | Functions for calculating vector norms, distances, normalization, and common operations in linear algebra and machine learning. |
 | [Bitmap](/sql-reference/functions/bitmap-functions) | Functions for bitmaps.                                                                                                          |
 | [Math](/sql-reference/functions/math-functions)  | Mathematical functions.                                                                                                         |
+| [Financial](/sql-reference/functions/financial-functions) | Financial functions.                                                                                                            |
```

Comment:
Not sure if that's the way to add new sections...

Reply:
This will link from the landing page to the new function page but won't add the page to the sidebar. This is done automatically. See my comment https://github.com/ClickHouse/ClickHouse/pull/81599#discussion_r2156473198